app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -479,6 +571,26 @@ class HomeActivityTest {
     return HomeActivity.createHomeActivity(context, profileId)
   }
 
+  private fun isEllipsized() = object : TypeSafeMatcher<View>() {
+    override fun describeTo(description: Description) {
+      description.appendText("with ellipsized text")
+    }
+    override fun matchesSafely(view: View): Boolean {
+      if (view !is TextView) {
+        return false
+      }
+      val textView: TextView = view
```

Comment:
This shouldn't be needed--the type check is enough for Kotlin to let you use ``view`` as a ``TextView``.

Reply:
Done